spec/integration/runner/variant-reassignment-example-10.spec.js

```diff
@@ -0,0 +1,98 @@
+ * |           |                                                            |                                                                         |                    | variants: { sku: v2, attributes: [ { brandId: 1 } ] }     |
+ * +-----------+------------------------------------------------------------+-------------------------------------------------------------------------+                    +-----------------------------------------------------------+
+ * |           |                                                            | Product:                                                                |                    |                                                           |
+ * |           |                                                            | id: 2                                                                   |                    |                                                           |
+ * |           |                                                            | slug: { en: "product-2" }                                               |                    |                                                           |
+ * |           |                                                            | product-type: "pt1"                                                     |                    |                                                           |
+ * |           |                                                            | masterVariant: { sku: v2, attributes: [ { brandId (sameForAll): 3 } ] } |                    |                                                           |
+ * +-----------+------------------------------------------------------------+-------------------------------------------------------------------------+--------------------+-----------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  const logger = utils.createLogger(__filename)
+  let ctpClient
+  let product1
+  let product2
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+    const productType = await utils.ensureProductType(ctpClient)
+    const productDraft1 = utils.generateProduct(['1'], productType.id)
+    productDraft1.masterVariant.attributes = [{ name: 'brandId', value: '2' }]
+    product1 = await utils.ensureResource(ctpClient.products, productDraft1)
+
+    const productDraft2 = utils.generateProduct(['2'], productType.id)
+    productDraft2.masterVariant.attributes = [{ name: 'brandId', value: '3' }]
+    product2 = await utils.ensureResource(ctpClient.products, productDraft2)
+  })
+
+  after(() =>
+    utils.deleteResourcesAll(ctpClient, logger)
+  )
+
+  it('merge variants with different sameForAll attributes without blacklist',
+    async () => {
+      const reassignment = new VariantReassignment([], logger, {})
+      await reassignment.execute([{
+        productType: {
```

Comment:
I remove all unnecessary attrs and the only repetitive attr is name. So far I think it would make it harder to read if I move it to separate function, so I will keep it like this? If it will be more repetitive, then I will move it to common place.

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/8/commits/24cb0d4e0b28f27d512e2f9b4b42aee080b32d4c